### DISCUSSION TITLE
Sideband 0.2.0 Released

### DISCUSSION THREAD

**Discussion by markqvist** [markqvist/Reticulum#110] [Announcements]
I have really been looking forward to bringing you all this major update to Sideband. There's lots of improvements and fixes on both Android and desktop, so without further ado, here is the changelog. With this release, Sideband becomes much more accessible and useful for general use. Hope you like it!

**Changes**
- Added persistent background service on Android
- Added notifications on Android and desktop
- Added ability to export and import LXMF identity keys
- Added Light/dark theme support
- Added full macOS support via pip
- Added build system for macOS application
- Significantly improved user interface
- Significantly improved backend for message and network handling
- Improved install and start on Linux
- Improved guide section
- Improved settings section
- Improved startup when TCP peer is unreachable
- Added preliminary USB support on Android
- Implemented app lifecycle management on Android
- Improved data persistence
- Added verbose logging output mode on desktop
- Added proper Android versioning codes
- Updated included RNS to 0.3.13 and LXMF to 0.1.9

![Screenshot](https://github.com/markqvist/Sideband/raw/main/docs/screenshots/devices_small.webp)

**Comment by HaosGames** [comment]
For what hardware is the preliminary USB support on Android intended? Because I have Heltec boards with recent rns firmware and the switch in the app to activate RNode Connectivity is still greyed out if I connect them. Maybe it's not yet ready for those boards?

**Reply by markqvist** [reply]
It will work for those boards too, but it is not quite there yet. I am still working on the last bits of the USB driver. So hang tight, we are almost there :)

Right now I am optimising the UI responsiveness a bit, and then I will return to work on the USB driver.
